src/main/java/org/jboss/tools/intellij/openshift/tree/application/ApplicationTreeModel.java

```diff
@@ -180,7 +180,7 @@ public Project getProject() {
             if (!node.isLoaded()) {
                 node.load();
             }
-            return Collections.list((Enumeration) ((MutableTreeNode)o).children());
+            return Collections.list(((MutableTreeNode)o).children());
```

Comment:
Don't remove the cast it's required for Java11

Reply:
done.